lib/ble_manager.dart

```diff
@@ -2,6 +2,13 @@ part of flutter_ble_lib;
 
 typedef RestoreStateAction = Function(List<Peripheral> restoreStateIdentifier);
 
+enum LogLevel { NONE, VERBOSE, DEBUG, INFO, WARNING, ERROR }
```

Comment:
Perhaps `peripheral_connection_state.dart` should be renamed  to `enums.dart` and contain this?